jekyll/_cci2/deployment-integrations.adoc

```diff
@@ -88,11 +88,11 @@ workflows:
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.4-node
+      - image: cimg/ruby:2.4-node
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:9.4.12-alpine
+      - image: cimg/postgres:9.4.12-alpine
```

Comment:
This image doesn't exist - https://circleci.com/developer/images/image/cimg/postgres

Reply:
`9.6` is a tag

Reply:
@felicianotech I went ahead and replaced the this with 9.6. Is that correct?